cmd/vulcan-aws-alerts/rds.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/adevinta/vulcan-report"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/sirupsen/logrus"
+
+	"github.com/adevinta/vulcan-check-sdk/state"
+)
+
+func caCertificateRotation(opt options, target string, vulcanAssumeRoleEndpoint string, roleName string, logger *logrus.Entry, state state.State) error {
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String("eu-west-1"),
+	})
+	if err != nil {
+		return err
+	}
+
+	creds, err := getCredentials(vulcanAssumeRoleEndpoint, target, roleName, logger)
+	if err != nil {
+		return err
+	}
+
+	// Iterate over all AWS regions where RDS is available
+	for region := range endpoints.AwsPartition().Services()[endpoints.RdsServiceID].Regions() {
+		sess.Config.Region = aws.String(region)
+		s := rds.New(sess, &aws.Config{Credentials: creds})
+
+		ctx := context.Background()
+		logger.Info(fmt.Sprintf("Describing pending maintenance actions on RDS for %s region", region))
+		result, err := s.DescribePendingMaintenanceActionsWithContext(ctx, &rds.DescribePendingMaintenanceActionsInput{})
+		if err != nil {
+			logger.Error(err)
+			continue
+		}
+		// TODO: implement pagination
+		if result.Marker != nil {
+			logger.Warn("DescribePendingMaintenanceActionsWithContext returned more than 1 page")
+		}
+
+		rg := report.ResourcesGroup{
+			Name:   `Rotation of CA certificate`,
+			Header: []string{"identifier", "account", "region", "dbname", "engine", "arn", "AutoAppliedAfterDate", "CurrentApplyDate"},
+			Rows:   []map[string]string{},
+		}
+
+		v := report.Vulnerability{
+			Summary:     `Rotation of CA certificate`,
+			Description: `Rotation of CA certificate`,
```

Comment:
Here I would use:
```
Due to the expiration of the AWS RDS CA, and to prevent downtime in your applications, you should add the new CA to your clients using a managed (i.e. RDS or Aurora) database through SSL/TLS and perform maintenance on the affected database instances before the certificate expiration date.
```